tensorflow/compiler/xla/service/gpu/cudnn_conv_rewriter.cc

```diff
@@ -529,7 +538,7 @@ StatusOr<bool> RunOnInstruction(HloInstruction* conv) {
     }
 
     return nullptr;
-  }();
+  }().ValueOrDie();
```

Comment:
This crash can be triggered by valid user input. In this case we want to propagate the status back to the user instead.

Reply:
I have made another change by pulling out this lambda as a regular helper function and call it with TF_ASSIGN_OR_RETURN to return error status back to the caller.